### PR TITLE
SNOW minor wording & docs URL updates

### PR DESCRIPTION
**Pull request by vadimkerr**
Related to https://github.com/grafana/oncall-private/issues/2615

## Checklist

- [x] Unit, integration, and e2e (if applicable) tests updated
- [x] Documentation added (or `pr:no public docs` PR label added if not required)
- [x] Added the relevant release notes label (see labels prefixed w/ `release:`). These labels dictate how your PR will
    show up in the autogenerated release notes.

### CODE REVIEW

File: grafana-plugin/src/containers/ServiceNowConfigDrawer/ServiceNowConfigDrawer.tsx [hunk 1]
```diff
@@ -111,24 +110,6 @@ export const ServiceNowConfigDrawer: React.FC<ServiceNowConfigurationDrawerProps
...
-
-                <Text>
-                  Description for such object and{' '}
-                  <a href={'#'} target="_blank" rel="noreferrer">
-                    <Text type="link">link to documentation</Text>
-                  </a>
-                </Text>
-              </VerticalGroup>
-            </div>
-
```
Comment (vadimkerr) [inline grafana-plugin/src/containers/ServiceNowConfigDrawer/ServiceNowConfigDrawer.tsx:131]: This is not a thing atm, so removing it completely (so settings are easier to navigate)

**Review by matiasb — APPROVED** [review]
LGTM

File: grafana-plugin/src/pages/integration/OutgoingTab/ConnectedIntegrationsTable.tsx [hunk 2]
```diff
@@ -84,11 +84,18 @@ const ConnectedIntegrationsTable: FC<ConnectedIntegrationsTableProps> = observer
                 <HorizontalGroup>
                   <Text type="secondary">Backsync</Text>
                   {tokenExists ? (
-                    <Tooltip content={<>Switch on to start sending data from other integrations</>}>
+                    <Tooltip
+                      content={
+                        <>
+                          Switch on to receive updates from ServiceNow. If disabled, Grafana OnCall will still send
+                          alerts to ServiceNow, but not receive updates back.
```
Comment (teodosii) [inline grafana-plugin/src/pages/integration/OutgoingTab/ConnectedIntegrationsTable.tsx:91]: Slight correction
- Switch on to receive updates from ServiceNow. If disabled, Grafana OnCall will still send alerts to ServiceNow, but **will** not receive **any** updates back.

File: grafana-plugin/src/containers/IntegrationForm/IntegrationForm.tsx [hunk 3]
```diff
@@ -346,6 +356,16 @@ export const IntegrationForm = observer(
                 <div className={styles.webhookSwitch}>
                   <Switch value={field.value} onChange={field.onChange} />
                   <Text type="primary"> Create default outgoing webhooks</Text>
+                  <Tooltip
+                    content={
+                      <>
+                        If enabled, all the necessary webhooks will be created automatically. It's highly recommended to
+                        keep this enabled.
```
Comment (teodosii) [inline grafana-plugin/src/containers/IntegrationForm/IntegrationForm.tsx:363]: Up to you but seems more appropriate in an enterprise context to refer exactly to **this option**
`It's highly recommended to keep this option enabled.`

**Review by teodosii — APPROVED** [review]

File: grafana-plugin/src/pages/integration/OutgoingTab/ConnectedIntegrationsTable.tsx (hunk 2 above)
Comment (vadimkerr) [reply grafana-plugin/src/pages/integration/OutgoingTab/ConnectedIntegrationsTable.tsx]: https://github.com/grafana/oncall/pull/4229/commits/41f1cba784fa9217e53a8a2269ebd6a3922f1ef2

File: grafana-plugin/src/containers/IntegrationForm/IntegrationForm.tsx (hunk 3 above)
Comment (vadimkerr) [reply grafana-plugin/src/containers/IntegrationForm/IntegrationForm.tsx]: https://github.com/grafana/oncall/pull/4229/commits/41f1cba784fa9217e53a8a2269ebd6a3922f1ef2

File: grafana-plugin/src/containers/IntegrationForm/IntegrationForm.tsx [hunk 4]
```diff
@@ -346,6 +356,16 @@ export const IntegrationForm = observer(
...
                   <Text type="primary"> Create default outgoing webhooks</Text>
+                  <Tooltip
+                    content={
+                      <>
+                        If enabled, all the necessary webhooks will be created automatically. It's highly recommended to
+                        keep this option enabled.
+                      </>
+                    }
+                  >
+                    {<Icon name={'info-circle'} />}
```
Comment (brojd) [inline grafana-plugin/src/containers/IntegrationForm/IntegrationForm.tsx:367]: I think curly brackets are not needed here? They will be rendered on UI

Comment (vadimkerr) [reply grafana-plugin/src/containers/IntegrationForm/IntegrationForm.tsx]: https://github.com/grafana/oncall/pull/4229/commits/cbcf15773d66ac5929d68137bd55798a1b8c33e3

**Review by brojd — APPROVED** [review]

**Pull request merged**